graphql/match/mutations.ts

```diff
@@ -52,4 +53,17 @@ export class MutateMatchResolver {
         }
         return true;
     }
+
+    @Mutation((returns) => JSONResolver, { nullable: true })
+    @AuthorizedDeferred(Role.ADMIN, Role.OWNER)
+    async matchCreateAdHocMeeting(@Ctx() context: GraphQLContext, @Arg('matchId', (type) => Int) matchId: number) {
+        const { user } = context;
+        const match = await getMatch(matchId);
+        await hasAccess(context, 'Match', match);
+        if (user.pupilId) {
+            return;
+        }
```

Comment:
We should throw an AuthorizationError here, so that we get some context in the backend, if nothing happens